lib/compiler/passes/generate-bytecode.js

```diff
@@ -479,6 +493,7 @@ function generateBytecode(ast) {
           action: null
         }),
         buildCondition(
+          node.match | 0,
```

Comment:
what does this mean?

Reply:
No, it wasn't.  You gave a Zorba the Mystic "when you look deep enough into the soul of `match` you will understand."